app/containers/Applications/ApplicationDetailsContainer.tsx

```diff
@@ -39,6 +48,9 @@ export const ApplicationDetailsComponent: React.FunctionComponent<Props> = ({
   const diffFromResults = review
     ? diffQuery?.old?.orderedFormResults?.edges
     : undefined;
+  const attachments = applicationRevision.attachmentsByApplicationIdAndVersionNumber
+    ? applicationRevision.attachmentsByApplicationIdAndVersionNumber.edges
+    : undefined;
```

Comment:
Could use optional chaining to shorten this, something like `const attachments = applicationRevision?.attachmentsByApplicationIdAndVersionNumber?.edges`

And then instead of checking if `attachments` is false for conditional rendering, check if its length is 0